### PR TITLE
Add metrics and event report for Ray channels

### PR DESCRIPTION
**Pull request by chaokunyang**
## What do these changes do?
This PR add ray channel metrics and event report
## Related issue number

Fixes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass, see [here](https://docs.pymars.org/en/latest/development/contributing.html#check-code-styles) for how to run them

### CODE REVIEW

File: mars/tests/test_utils.py [hunk 1]
```diff
@@ -586,3 +586,8 @@ def __call__(self, *args, **kwargs):
 
     func = Func()
     assert get_func_token_values(func) == [func]
+
+
+@require_ray
+def test_report_event(ray_start_regular):
+    utils.report_event("WARNING", "test_label", "test_message")
```
Comment (wjsi) [inline mars/tests/test_utils.py:593]: No validations for this test, please change.

File: mars/utils.py [hunk 2]
```diff
@@ -1629,6 +1634,18 @@ def cache_tileables(*tileables):
             t.cache = True
 
 
+def report_event(severity, label, message):
```
Comment (wjsi) [inline mars/utils.py:1637]: Seems a ray-specific function, put into ray backend.

Comment (hekaisheng) [inline mars/utils.py:1637]: If it is only for Ray, put it into "mars/oscar/backends/ray/utils.py".

File: mars/utils.py [hunk 3]
```diff
@@ -73,6 +73,11 @@
 from .lib.version import parse as parse_version
 from .typing import ChunkType, TileableType, EntityType, OperandType
 
+try:
+    import ray
```
Comment (wjsi) [inline mars/utils.py:77]: Should use lazy import.

**Review by wjsi — CHANGES_REQUESTED** [review]

File: mars/utils.py (hunk 2 above)
Comment (chaokunyang) [reply mars/utils.py:1637]: The ray executor can use it to report event too.

Comment (hekaisheng) [reply mars/utils.py:1637]: Can import from "mars/oscar/backends/ray/utils.py".

File: mars/utils.py (hunk 3 above)
Comment (chaokunyang) [reply mars/utils.py:77]: The `lazy_import` is defined after this import, using `lazy_import` will make it not located at the top import statements.

File: mars/tests/test_utils.py (hunk 1 above)
Comment (chaokunyang) [reply mars/tests/test_utils.py]: Fixed it by mock report_event

File: mars/utils.py (hunk 2 above)
Comment (chaokunyang) [reply mars/utils.py]: moved to `mars/oscar/backends/ray/utils.py`

Comment (chaokunyang) [reply mars/utils.py]: moved to mars/oscar/backends/ray/utils.py

**Review by wjsi — APPROVED** [review]
LGTM

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**